Units/parser-gdscript.r/sample.d/expected.tags

```diff
@@ -0,0 +1,17 @@
+a	input.gd	/^var a = 5$/;"	v
+s	input.gd	/^var s = "Hello"$/;"	v
+arr	input.gd	/^var arr = [1, 2, 3]$/;"	v
+dict	input.gd	/^var dict = {"key": "value", 2: 3}$/;"	v
+typed_var	input.gd	/^var typed_var: int$/;"	v	typeref:typename:int
+inferred_type	input.gd	/^var inferred_type := "String"$/;"	v
+ANSWER	input.gd	/^const ANSWER = 42$/;"	v	typeref:typename:const
+THE_NAME	input.gd	/^const THE_NAME:String = "Charly"$/;"	v	typeref:typename:const String
+anon_enum	input.gd	/^enum {UNIT_NEUTRAL, UNIT_ENEMY, UNIT_ALL}$/;"	g
+Named	input.gd	/^enum Named {THING_1, THING_2, ANOTHER_THING=1}$/;"	g
+v2	input.gd	/^var v2 = Vector2(1, 2)$/;"	v
+v3	input.gd	/^var v3 = Vector3(1, 2, 3)$/;"	v
+some_function	input.gd	/^func some_function(param1, param2):$/;"	f
+something	input.gd	/^func something(p1, p2):$/;"	f
+Something	input.gd	/^class Something:$/;"	c
+a	input.gd	/^    var a = 10$/;"	v	class:Something
+_init	input.gd	/^func _init():$/;"	f
```

Comment:
Could you add a test case for method, module, enumerator, parameter, and local?

Reply:
I have lines that would be related to module, enumerator, parameter and local, not showing up in the test output though. Module is extends (kind of), enumerator is by the two enums, there's some functions with parameters and a `var local_var = 5` in `some_function`.  Can add a method to the class.

Reply:
Put
```
--kinds-GDScript=+{parameter}
--kinds-GDScript=+{local}
```
to the args.ctags file.